src/main/java/com/puppycrawl/tools/checkstyle/checks/coding/MissingSwitchDefaultCheck.java

```diff
@@ -55,6 +55,10 @@
  *     and expressions.
  * </p>
  * <p>
+ * See the <a href="https://docs.oracle.com/javase/specs/jls/se17/html/jls-14.html#jls-14.30">
```

Comment:
> new record pattern syntax introduced in Java 21

I would expect use to use the new JLS in which it was introduced. Update the one before it as well.

Reply:
done